### DISCUSSION TITLE
Welcome to Security-Compliance Discussions!

### DISCUSSION THREAD

**Discussion by its-a-lisa-at-work** [18F/Security-Compliance#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PhilTortora** [comment]
Hi All, I work at SafeGuard Cyber, a company that offers digital risk protection and compliance for cloud-based collaboration, chat, and social media platforms. I'm here to learn, not sell!

I'm curious as to what measures some of your companies are implementing to ensure compliance on channels outside your network perimeter? I'm also interested in learning about what your biggest compliance needs are in general?

Looking forward to seeing future conversations here!
